chain/epoch_manager/src/lib.rs

```diff
@@ -95,6 +99,7 @@ impl EpochManager {
         genesis_protocol_version: ProtocolVersion,
         reward_calculator: RewardCalculator,
         validators: Vec<ValidatorStake>,
+        additional_epochs_to_keep: u64,
```

Comment:
An allternative to adding an extra parameter to both construtors is to add a single builted-lite method: 

```rust
impl EpochManager {
    pub with_additional_epochs_to_keep(mut self, additional_epochs_to_keep: u64) -> EpochManager {
        self.additional_epochs_to_keep = additional_epochs_to_keep;
        self
    }
}
```

Reply:
@matklad Yes, I see, we can add it. Though, I think the the design would be signed to maybe only keep it in NightShade.